src/Illuminate/Support/Str.php

```diff
@@ -94,6 +101,21 @@ public static function camel($value)
         return static::$camelCache[$value] = lcfirst(static::studly($value));
     }
 
+    /**
+     * Convert a value to pascal case.
+     *
+     * @param  string  $value
+     * @return string
+     */
+    public static function pascal($value)
```

Comment:
The methods in this class are sorted alphabetically. Please move this method further down the class.